addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js

```diff
@@ -275,6 +275,9 @@ const PublicLivechatWindow = Widget.extend({
      * @private
      */
     _onScroll() {
+        if (!this.messaging.publicLivechatGlobal.livechatButtonView.chatWindow) {
```

Comment:
I'm not sure exactly what can exist or not in these new fields, but shouldn't it also guard for other parts of the relation?

Reply:
No need to guard other fields: `livechatButtonView` always exist in this flow, `publicLivechatView` is coupled with `chatWindow`, and `PublicLivechatView/widget` is coupled with the model `PublicLivechatView` itself.

Same remark with `publicLivechat` & `PublicLivechat/legacyPublicLivechat`

Reply:
But if the scroll is debounced we are potentially in no specific flow? For all we know even messaging itself could be deleted.

Reply:
Indeed, will do proper guard